src/index.md

```diff
@@ -8,6 +8,93 @@ styles:
 
 <div class="content-sizing" markdown="1">
 
+# Overview
+
+Workbox is a collection of libraries and build tools that make it easy to
+build web apps that work offline.
+
+Modern offline web apps are possible thanks to **service workers** (SWs). SWs
+are background workers, written in JavaScript, that can locally save all of
```

Comment:
Maybe link up background workers to: https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API